README.md

```diff
@@ -1,32 +1,31 @@
-# Show Whitespace in Brackets
+# Show Whitespace
 
-[Brackets](http://brackets.io/) extension to visualize whitespace (spaces and tabs) like [Sublime Text](http://www.sublimetext.com/) does. Adds an entry called "Show Whitespace" to the View menu, the shortcut is Command-Shift-W (or Ctrl-Shift-W on Windows). The state is remembered next time you start Brackets.
+> A [Brackets](http://brackets.io/) extension to visualize whitespace (both spaces and tabs) in the same style as [Sublime Text](http://www.sublimetext.com/).
 
-This extension overwrites `Line.getHTML` (part of [CodeMirror](http://codemirror.net/)) and modifies its output. Spaces in the indentation are wrapped as `<span class="cm-space dk-leading"> </span>`, normal spaces as `<span class="cm-space dk-normal"> </span>`. CodeMirror itself already handles tabs similarly, this extension turns those into `<span class="cm-tab dk-leading"> </span>` if they are part of the whitespace starting a line and `<span class="cm-tab dk-normal"> </span>` otherwise.
+## Usage
 
-The styles are defined in `main.less` which is compiled to CSS and loaded into Brackets when loading the extension.
+Activate using `View > Show Whitespace` or by using the shortcut `Ctrl-Shift-W` (`Command-Shift-W` on Mac OS X). The extension state is remembered across Brackets launches. Whitespace in all editor windows are visualized, including inline editors.
 
-Whitespace in inline editors is visualized as well.
+## Technical Details
 
+This extension uses [CodeMirror](http://codemirror.net/) overlays to construct and render whitespace for both spaces and tabs. Each individual blank character are wrapped in a `<span></span>` with an appropriate denoting class. The following are all the classes this extension uses and their purpose. Do note that tabs follow this same structure, with the word `space` replaced with `tab`.
 
-## Install
-
-Open Brackets and click Help > Show Extensions Folder
-Clone the extension into the user subdirectory:
-
-    git clone git://github.com/DennisKehrig/brackets-show-whitespace.git user/ShowWhitespace
+* `.cm-dk-whitespace-space`: Whitespace between any characters
+* `.cm-dk-whitespace-leading-space`: Whitespace that makes up any indentation
+* `.cm-dk-whitespace-empty-line-space`: Any lines that consist solely of whitespace
+* `.cm-dk-whitespace-trailing-space`: Whitespace at the end of a line after any characters
 
+The primary extension styling is defined in `styles/main.less`, which is compiled into CSS, while whitespace colors are defined in Brackets preferences and rendered into `styles/whitespace-colors-css.tmpl`. Both files are then loaded into Brackets on startup.
 
-## Changelog
-
-### 2012-11-14
+## Install
```

Comment:
I can see that.  I guess I like consistency... If every extension pointed to the link then you wouldn't see little variations of the same stuff... :)  oh well.